lib/cloud_controller/diego/ssh_key.rb

```diff
@@ -21,7 +22,9 @@ def authorized_key
           end
         end
 
-        delegate :fingerprint, to: :key
+        def fingerprint
+          ::SSHKey.new(key.to_der).sha1_fingerprint
```

Comment:
Small remark: I would rather cache the fingerprint in case someone decides to call this method twice (which doesn't seem to be the case today, but the `authorized_key` is also cached), i.e.
```
@fingerprint ||= ...
```